uh_internal/apps/core/backends.py

```diff
@@ -54,6 +60,69 @@ def authenticate(self, ticket, service, request):
             else:
                 principal_name = str(user_info["userPrincipalName"])
 
+                user_group_objects = Reader(connection=connection,
+                                            object_def=account_def,
+                                            query='(&(member=CN=kjreis,OU=People,OU=Enterprise,OU=Accounts,DC=ad,DC=calpoly,DC=edu)(objectClass=group))',
```

Comment:
this probably needs to be non-specific to you

Reply:
whoops, and this is why I requested a code review.